<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MtbUniIOWindow</class>
 <widget class="QDialog" name="MtbUniIOWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_inputs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_inputs"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_outputs">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>271</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Outputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_outputs"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
